frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/ModelManager.java

```diff
@@ -193,4 +210,12 @@ public boolean scaleRequestStatus(String modelName) {
     public void submitTask(Runnable runnable) {
         wlm.scheduleAsync(runnable);
     }
+
+    public ConcurrentLinkedDeque<Model> getStartupModels() {
+        return startupModels;
+    }
+
+    public void setStartupModels(Model model) {
```

Comment:
Should not expose setter.

Reply:
Ya, true. We shouldn't expose public setter.